tests/e2e/machineregistration_test.go

```diff
@@ -23,54 +23,60 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	http "github.com/rancher-sandbox/ele-testhelpers/http"
-	elementalv1 "github.com/rancher/elemental-operator/pkg/apis/elemental.cattle.io/v1beta1"
+
 	"github.com/rancher/elemental-operator/pkg/config"
-	corev1 "k8s.io/api/core/v1"
-	rbacv1 "k8s.io/api/rbac/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	"github.com/rancher/elemental-operator/tests/catalog"
+
+	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
 )
 
 var _ = Describe("MachineRegistration e2e tests", func() {
 	Context("registration", func() {
-		var mRegistration *elementalv1.MachineRegistration
+		var mRegistration string = "machine-registration"
+		k := kubectl.New()
```

Comment:
Why use kubectl when we can use controller-runtime client? we can have better control over API objects when writing tests.